packages/core/lib/src/styles/app_themes.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:core/src/styles/app_colors.dart';
+import 'package:core/src/styles/app_text_styles.dart';
+import 'package:flutter/material.dart';
+
+/// Helper class for app wide themes
+class AppThemes {
+  /// Returns theme data
+  static ThemeData getTheme({bool isDark = false}) {
```

Comment:
This will likely be a problem with the `widgetbook_generator`, but I think we should adjust the generator to be able to handle `static` getter and not this code.